### ISSUE TITLE
[docdb] org.yb.cql.TestInsertJson#ttl is flaky in master

### ISSUE THREAD

**Issue by bmatican** [yugabyte/yugabyte-db#3030]
See https://detective-gcp.dev.yugabyte.com/job/github-yugabyte-db-centos-master-gcc-debug/499/artifact/java/yb-cql/target/surefire-reports_org.yb.cql.TestInsertJson__ttl/org.yb.cql.TestInsertJson-output.txt

> org.junit.ComparisonFailure: expected:<[Row[0, Hello!]]> but was:<[]>

**Comment by bmatican** [comment]
This is top 50 in the stability endpoint now

**Comment by frozenspider** [comment]
[Here's this test logs archive](https://github.com/yugabyte/yugabyte-db/files/4711785/logs.zip) from the following Jenkins builds:
```
centos-master-clang-asan #962
centos-master-clang-debug #939
centos-master-clang-release #948
centos-master-clang-tsan #959
centos-master-gcc-debug #929
mac-master-clang-debug #1230
```

**Comment by frozenspider** [comment]
Temporary fix introduced in a8b497c; should be reverted after #4663
